### ISSUE TITLE
Add options to preload modules

### ISSUE THREAD

**Issue by HansPinckaers** [pappasam/coc-jedi#5]
Hi there,

Is it possible to add an option to preload certain modules, so that autocompletion feels faster? For example, useful for big libraries such as numpy.

See jedi docs: https://jedi.readthedocs.io/en/latest/docs/api.html#jedi.preload_module 

Thanks!
Hans

**Comment by pappasam** [comment]
In the latest release, I've handled this by exposing `auto_import_modules` to the end user. See here: https://github.com/pappasam/coc-jedi#jedijedisettingsautoimportmodules

Please let me know if this solution performs acceptably for you.

**Comment by HansPinckaers** [comment]
Hi! Awesome!

Sadly however it doesn't seem to work? At least the first time completions of libraries I add is not sped up.

I wanted to debug myself, but could not find a reference to jedi's preload_module, only to auto_import_modules. Reading the documentation I'm not sure auto_import_modules is the same as preload_module.

**Comment by pappasam** [comment]
@HansPinckaers please install the latest version of `jedi-language-server`. I suspect that you only updated `coc-jedi`.

In the future, until we're able to figure out how to set up a nice environment automatically for conda users, you'll need to manually update `jedi-language-server`.

**Comment by pappasam** [comment]
Additionally, you are correct that `preload_module` is different than the `auto_import_modules`. Based on my testing, `preload_module` didn't yield substantial speed improvements while `auto_import_modules` sped things up considerably.

It's still not perfect, but it made numpy completions bearable (at least on my machine).

**Comment by HansPinckaers** [comment]
After a first test, I did indeed remember to also update the `jedi-language-server` :slightly_smiling_face: 

For me, numpy autocomplete is pretty fast (also with coc-python), I do not really see any benefit when using `auto_import_modules`. However, the first time is slow, that is why I wanted to try `preload_module`. 

I tried to hack it in like this, but it does not seem to do improve the first autocomplete. I wonder if it is a bug of jedi in that case.

![Screenshot from 2020-05-30 14-11-29](https://user-images.githubusercontent.com/70747/83327973-3233f180-a280-11ea-8e01-2ec51c42885b.png)
(I created a `preload_module` function in jedi_utils that just calls `jedi.preload_module`)
